### DISCUSSION TITLE
Ability to send reference number as the description for gateways, instead of order ID

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/commerce#2349] [Ideas]
When using a custom reference number for orders, it'd be great if you could push this as part of the description to Stripe, PayPal and other payment gateways in the description. The would help clients in associating orders with their transactions without having to step through the order element's ID first (which they wouldn't be using).

This might be tricky because the reference number is only calculated once the order is complete, so not sure on the best approach here.

**Comment by gojira32** [comment]
Yep, this is causing our customer service guys a lot of hassle not being able to use the order reference # in order to search for payments directly in Stripe and PayPal.
